app/overlays/tgl/ipc4_overlay.conf

```diff
@@ -47,3 +47,6 @@ CONFIG_DMA_DW_FIFO_PARTITION=y
 CONFIG_DMA_INTEL_ADSP_GPDMA_HAS_LLP=y
 
 CONFIG_HEAP_MEM_POOL_SIZE=8192
+
+CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
+CONFIG_GOOGLE_RTC_AUDIO_PROCESSING_MOCK=y
```

Comment:
We used to have Chrome overlay for XTOS builds. This looks like something that should go to the Chrome overlay, because this ipc4_overlay.conf is the configuration we ship to upstream Linux distros and having a RTC mock included does not sound correct.

Reply:
sure, removed it from ipc4_overlay.conf.   It will be added in chrome overlay.